modules/xmpp/ChatRoom.js

```diff
@@ -578,7 +578,7 @@ ChatRoom.prototype.getMemberRole = function (peerJid) {
 
 ChatRoom.prototype.setJingleSession = function(session){
     this.session = session;
-    this.session.room = this;
+    this.session.initialize(false /* initiator */, this /* room */);
```

Comment:
Maybe we should move "session.initialize" call somewhere else. I don't think it's related to the ChatRoom instance. Maybe in JitsiConference or in JingleSession?